docs/devGuide/migratingToTypeScript.md

```diff
@@ -66,41 +65,27 @@ We have decided on structuring the commits as two commits: a "Rename" commit and
-   Named export   | `module.exports = { x }`   | `export = { x }`                                                 | `export { x }`
-   Default export | `module.exports = x`       | `export = x`                                                     | `export default x`
-
-   Similar to the process of refactoring a function, you might have to **adjust how these files are imported by other TypeScript (`.ts`) files**. If you are changing it to the TypeScript equivalent syntax, then it must be imported with so as well. The same goes for the ES6 syntax. 
-
-   Start with the TypeScript equivalent syntax, as it is compiled to be exactly the same as the CommonJS syntax (except the named import case), making it the safest choice when adapting the file. You can read more about this syntax in the [official documentation](https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require).
+2. **Change the CommonJS module import/export syntax** to TypeScript's equivalent syntax (use ES6 syntax only if possible).
 
-   The ES6 syntax compiles into something more than just the CommonJS syntax when compiled, and might lead to unexpected behaviours when another `.js` file imports the compiled file, to the point of failing tests in some cases. You can try changing to the ES6 syntax, but only proceed if there are no failing tests. The TypeScript equivalent syntax is enough for the initial adaptation.
+   A common error in newly renamed files comes from the way modules are imported/exported. Those statements can be converted according to the [Import/Export Syntax Reference](#importexport-syntax-reference) section below. The sequence is as follows:
 
-   However, in some cases, changing to the ES6 syntax might be necessary. For instance, you cannot use the TypeScript equivalent syntax if you want to import/export TypeScript specific constructs, such as types, interfaces, and enums. Again, when doing so, make sure that all tests pass.
+   1. Start by changing the import/export statements into TypeScript equivalent syntax. Be careful with the imports as you have to match them with the export syntax that the modules use, be it TypeScript equivalent or ES6.
 
-   <box type="info">
+   2. Check whether it is possible to change the export use ES6 syntax. As mentioned in the syntax reference, ES6 should not be used if the module only exports a single thing. Change to ES6 syntax only if possible, otherwise keep with TypeScript equivalent syntax.
 
-   We will perform a mass change into ES6 syntax after all files in core have been adapted into TypeScript.
-   </box>
+   3. You might have to **adjust how the files are imported by other TypeScript (`.ts`) files**. Before, you would see that some If you are changing the export to the TypeScript equivalent syntax, then it must be imported the same way as well. The same goes for the ES6 syntax.
```

Comment:
> You might have to **adjust how the files are imported by other TypeScript (`.ts`) files**. Before, you would see that some If you are changing the export to the TypeScript equivalent syntax, then it must be imported the same way as well. The same goes for the ES6 syntax.

> You might have to **adjust how the files are imported by other TypeScript (`.ts`) files**. ~Before~Previously, you would see that ~some~ if you are changing the export to the TypeScript equivalent syntax, then it must be imported the same way as well. The same goes for the ES6 syntax.

Reply:
Some excess statement there 😅 removing the `Before, you would see that some` part instead.